.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,50 @@
+name: Ray feature request
+description: Suggest an idea for Ray project
+title: "[Feature] "
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for finding the time to propose new feature!
+        We really appreciate the community efforts to improve Ray.
+  - type: checkboxes
+    attributes:
+      label: Search before asking
+      description: >
+        Please make sure to search in the [issues](https://github.com/ray-project/ray/issues) first to see
+        whether the same feature was requested already.
+      options:
+        - label: >
+            I had searched in the [issues](https://github.com/ray-project/ray/issues) and found no similar
+            feature requirement.
+          required: true
+  - type: textarea
+    attributes:
+      label: Description
+      description: A short description of your feature
+
+  - type: textarea
+    attributes:
+      label: Use case
+      description: What do you want to happen?
```

Comment:
```suggestion
      description: Describe the use case of your feature request. It will help us understanding and prioritizing the feature request.
```